speedy-recovery-client/src/components/secured/shared/Header.js

```diff
@@ -78,6 +87,24 @@ class Header extends Component {
       </header>
     );
   }
+
+  componentDidMount() {
+    if (this.props.role !== "Patient") {
+      let id = this.props.userid;
+      getConversation(id)
+            .then(value => {
+              var unreadNum = 0;
+              for (let i = 0; i < value.length; i ++){
```

Comment:
use es6 for of loop here

Reply:
es6 for of loop and the normal for loop both work here. Is there any specific reason that I have to use es6 for of loop?

Reply:
for of loop is a new feature in es6 and it has been used in all other files too. So to be consistent you should use it

Reply:
Ok, I'll change it here.

Reply:
It's generally better readable